docs/features.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Supported features
+sidebar_label: Features
+---
+
+Margarita is heavily inspired by our <a href="http://kiwi.com" target="_blank">Kiwi.com</a> mobile and web apps. We want to create full-fledged production ready app for developers, entrepreneurs and companies who want to use our API.
+
+## Supported Features
+
+Margarita is separated to 4 main parts of the app:
```

Comment:
Margarita is split into 4 parts or functional domains:

Reply:
Done